src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceFacadeImpl.java

```diff
@@ -144,6 +149,22 @@
         }
     }
 
+    private void validateServiceRoleMappingSheet(List<CaseWorkerDomain> caseWorkerRequestList,
+                                                 Class<? extends CaseWorkerDomain> classType) {
+        if (nonNull(classType) && classType.equals(ServiceRoleMapping.class)) {
+            boolean multipleServiceCode = caseWorkerRequestList.stream()
+                    .filter(ServiceRoleMapping.class::isInstance)
+                    .map(ServiceRoleMapping.class::cast)
+                    .map(ServiceRoleMapping::getServiceId)
+                    .filter(StringUtils::isNotEmpty)
```

Comment:
If any serviceId is empty, the JSR will find it. Hence skipping those records from here.